operator/src/main/java/oracle/kubernetes/operator/work/Fiber.java

```diff
@@ -571,6 +572,31 @@ public Packet getPacket() {
     return na.getPacket();
   }
 
+
+  @Override
+  public void cancelChildFibersAndResume(Packet packet, Step next) {
+    na.setNext(next);
+    if (children != null) {
+      for (Fiber child: children) {
+        child.cancelAndExitCallback(true, NULL_EXIT_CALLBACK);
```

Comment:
I notice that this assumes that this method doesn't need to evaluate the return value from cancelAndExitCallback on each child. Did you evaluate this and decide that this wasn't necessary? So, thinking that through... After these calls we know that each child Fiber will be cancelled; however, it is possible that one or more child Fibers is still executing any Step that it was already executing. Can you see a use case where this would be a problem?

Reply:
The places where the return value is used need to count how many child fibers are still running, in order to allow the parent to exit gracefully. In this method, we aren't interested in the result of the child fibers, or in canceling the parent fiber right now. Essentially, we are aborting the make-right and seeking only to report a failure.